cloud-aws/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/util/AwsCloudFormationErrorMessageProviderTest.java

```diff
@@ -69,6 +71,7 @@ void shouldReportAddressLimitExceededOnCreate() {
                         .withResourceStatus(ResourceStatus.CREATE_COMPLETE.toString())
                         .withResourceStatusReason("Created")
         ));
+        when(cfRetryClient.describeStackEvents(any(DescribeStackEventsRequest.class))).thenReturn(new DescribeStackEventsResult());
```

Comment:
I think you should add an event here or add a new test for it.

Reply:
as the result is logged only and not returned for the UI it would be possible to test only if I move it to a separate class. But in that case all the others should be moved to a separate class. If you think it worth the effort I will do it.

Reply:
I see, then you can leave it as is.